Source/OIDTokenUtilities.m

```diff
@@ -48,4 +48,9 @@ + (NSData *)sha265:(NSString *)inputString {
   return sha256Verifier;
 }
 
++ (NSString *)truncate:(NSString *)inputString {
+  return inputString.length >= 9
```

Comment:
Can you convert this non-trivial multi-line ternary statement into a regular if-statement?